lsstlaunch.bash

```diff
@@ -1,19 +1,55 @@
 #!/bin/bash
 CONFIG_FILE=$1
-# /opt/lsst/sal only exists in SAL builds of the stack
-if [ -d /opt/lsst/sal ]; then
-    source /opt/lsst/sal/salbldsteps.bash 2>&1 > /dev/null
-    for i in xml idl sal salobj ATDome ATDomeTrajectory ATMCSSimulator \
-            simactuators standardscripts scriptqueue externalscripts ; do
-        setup ts_${i} -t current
-    done
+
+fallback () {
+    # /opt/lsst/sal only exists in SAL builds of the stack
+    if [ -d /opt/lsst/sal ]; then
+        source /opt/lsst/sal/salbldsteps.bash 2>&1 > /dev/null
+        for i in xml idl sal salobj ATDome ATDomeTrajectory ATMCSSimulator \
+                simactuators standardscripts scriptqueue externalscripts ; do
+            setup ts_${i} -t current
+        done
+	setup lsst_sitcom  # RFC-992
+    else
+        source /opt/lsst/software/stack/loadLSST.bash
+        setup lsst_distrib
+    fi
+    setup display_firefly
+}
+
+if [ -z "${RSP_SITE_TYPE}" ]; then
+   fallback  # Preserve backwards compatibility for a while
 else
-    source /opt/lsst/software/stack/loadLSST.bash
-    setup lsst_distrib
+    case ${RSP_SITE_TYPE} in
+	telescope)
+            source /opt/lsst/sal/salbldsteps.bash 2>&1 > /dev/null
+            for i in xml idl sal salobj ATDome ATDomeTrajectory \
+		     ATMCSSimulator simactuators standardscripts \
+		     scriptqueue externalscripts ; do
+		setup ts_${i} -t current
+            done
+	    for i in lsst_sitcom firefly; do
+		setup $i
+	    done
```

Comment:
Why firefly and not display_firefly?

Also, could you unwind this loop for right now? It's only two commands, so making that into a three-line loop makes the code more complicated and harder to follow and I don't think accomplishes much.

Reply:
Good catch, should be display_firefly here and elsewhere.

Reply:
Loop unrolled here and elsewhere.